src/get-started/java-binding.md

```diff
@@ -503,6 +503,13 @@ Currently Java Binding includes the following built-in services:
   See the [*Anchoring Service*](../advanced/bitcoin-anchoring.md)
   article for more details.
 
+- **Time Oracle.**
+  Time oracle allows determining time, importing it from the external world to
```

Comment:
I don't like this description at all (I see where it's been copied from, sooo @Maria-Nosyk , @slowli , WDYT?). What can I do with this oracle? 

Allows other/user services to determine the current time?

Reply:
Agree. Updated this, feel free to propose another change in case the new version is still inadequate.

Reply:
It's good, thanks! @Maria-Nosyk , @slowli you might consider adopting it for the main article :sunglasses: